katas/scrabble-score/test.js

```diff
@@ -0,0 +1,70 @@
+function scrabbleScore(word) {
```

Comment:
Hi Reena, 
I have made the modifications you suggested to the `test.js` file. Let me know if there is anything else you would like to have changed. Thanks, Rhys 😄